toolbox

```diff
@@ -712,9 +712,7 @@ create()
             --volume "$HOME":"$HOME":rslave \
             --volume "$XDG_RUNTIME_DIR":"$XDG_RUNTIME_DIR" \
             --volume "$dbus_system_bus_path":"$dbus_system_bus_path" \
-            --volume /dev/bus:/dev/bus \
-            --volume /dev/dri:/dev/dri \
-            --volume /dev/fuse:/dev/fuse \
+            --volume /dev:/dev \
```

Comment:
It seems podman does not honor this. I'm getting a reduced `/dev` with this version of toolbox.

I suspect that `/dev` is correctly bind mounted, but later on, podman is mounting a reduced `/dev` over it.

Reply:
oops, I did not check by creating a new toolbox image :/

Reply:
it actually works (you need to create a new toolbox for that), but the file is owned by nobody :(

Reply:
Indeed. This was specified above. You need to change permissions in the host operating system in order to access many of the files.